src/main/java/com/twilio/http/TwilioRestClient.java

```diff
@@ -50,7 +57,16 @@ public Response request(final Request request) {
         if (edge != null)
             request.setEdge(edge);
 
-        return httpClient.reliableRequest(request);
+        logRequest(request);
```

Comment:
What happens if there's no logger configured?

Reply:
If no logger configuration is provided log4j uses a default logging configuration that sets `ERROR` as the logging level displayed to the console. No logging output will be shown since nothing in the library is using log4j to log anything and `DEBUG` is a higher intLevel.